benches/rs_receiver.rs

```diff
@@ -7,28 +7,28 @@
 // except according to those terms.
 
 
-extern crate pnet;
+extern crate pnet_datalink;
```

Comment:
These benchmarks should not really be in the `benches/` directory, since they are not `#[bench]` benchmarks. This folder is special to cargo. However, it's fairly convenient to have cargo automatically compile it so it get's tested by the CI. I'll leave it in this directory for now.

Reply:
My logic for putting them there was "these are benchmarks, and `benches/` is a benchmarks folder". It's a shame you can't just `cargo run` them like normal benchmarks though.

Reply:
Well, they are not benchmarks really, they are normal programs. If we put them in `examples/` we could run them with `cargo run --example rs_receiver`

Reply:
That might be a good idea. What would you do with the Python/C benchmarks?